src/components/MultiHopTrade/hooks/useQuoteValidationErrors.ts

```diff
@@ -0,0 +1,78 @@
+import { useHopHelper } from 'components/MultiHopTrade/hooks/useHopHelper'
```

Comment:
should this file be tsx?

Reply:
It absolutely should be. https://github.com/shapeshift/web/pull/4889/commits/3c2dfb0db4a92e0f1a79c3401a5342d15ab69a42